packages/@react-types/label/src/index.d.ts

```diff
@@ -33,7 +33,7 @@ export interface SpectrumLabelProps extends SpectrumLabelPropsBase, HTMLAttribut
 export interface SpectrumFieldProps extends SpectrumLabelPropsBase, SpectrumHelpTextProps {
   children: ReactElement,
   label?: ReactNode,
-  labelProps: HTMLAttributes<HTMLElement>,
+  labelProps?: HTMLAttributes<HTMLElement>,
```

Comment:
I was surprised that labelProps didn't need a default set in Field, but seems like spreading undefined/null is ok if done in a object TIL

Reply:
yep, you can spread undefined/null all you like, plus it's already optional here https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/textfield/src/TextFieldBase.tsx#L30